### DISCUSSION TITLE
Add Andy Gavin Jason Rubin and Water111 into the base OG version.

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#3700] [Ideas]
Add them as custom actors behind the locked precursor door as the three sages of GOAL.

**Comment by Zedb0T** [comment]
Maybe a whole custom level that unlocks after beating the game to show the contributors?

**Comment by Zedb0T** [comment]
Or an insomniac museum style thing that shows off and documents some of the insides of development of openGOAL (and shows off fun bugs/easter eggs (saladman fingers fisherman))

**Reply by yodaxtah** [reply]
> Or an insomniac museum style thing that shows off and documents some of the insides of development of openGOAL

I find that a cool idea! Ignoring the work required to implement it for brainstorming, what about including the [milestones](https://opengoal.dev/progress/milestones/)? If there is much to say about getting each labeled milestone to work, that would be interesting to put each object/surface/rendered-thing in a moment, provided with a message to read / play. Maybe other devs that helped out can also provide read-aloud audio? If I recall correctly, in the museum of R&C 3, each developer explained their own contribution/project within the game.

Regarding the content, I'd personally be interested in the technical stuff, but the majority won't understand all that. So that accessible (i.e., non-technical) information can be easily found, the room can be split the room, or color codes can be used. To make it more “interesting”, don't forget to include objects that can be interacted with. E.g., you can add a Zoomer or Flut Flut in the museum, much like R&C's museum, or the canon, if there is something to say about it. Even if it is not directly related to the milestones, the decompilation allowed us to get a complete understanding of their mechanics/metrics and what's possible with them.

Regarding the implementation, I'd suggest to provide it as a mod which requires a game's 100% completion to open the door. In the game itself, you can "simply" reference the mod, without including it. This way, there is a separation between the jak-project, and potential problems that may follow from documenting stuff that may not be in the game. I can imagine that playing dumped unused audio, adding Gavin's/Ruben's face in the game or adding any other cool idea that is not directly tied to the game, but valuable in the museum, may be a small risk. And, you still have a vanilla experience, if anyone cares strongly about that.
